packages/backend/src/models/SchedulerModel/index.ts

```diff
@@ -444,6 +444,21 @@ export class SchedulerModel {
     }
 
     async logSchedulerJob(log: SchedulerLog): Promise<void> {
+        console.log({ log });
+        if (log.schedulerUuid) {
+            console.log('there is a scheduler uuid');
+
+            const schedulersList = await this.database(SchedulerTableName)
+                .select('scheduler_uuid')
+                .where('scheduler_uuid', log.schedulerUuid);
+
+            if (schedulersList.length === 0) {
+                console.log('schduler list empty');
+
+                return;
+            }
+        }
+
```

Comment:
replicated this scheduler list being empty @ZeRego , this is what the server logged: 
![image](https://user-images.githubusercontent.com/7611706/233629299-7315cb3b-2566-4db0-8395-7fa4bfea7d21.png)

```shell
2023-04-21 11:47:00 [Lightdash] error: Unhandled Rejection at Promise Scheduler not found
schduler list empty 
```

I'm assuming this error would be caught by Sentry?

Reply:
issue was to do with the `setTimeout` not being done properly when trying to replicate the scenario. Ignore the above